test/Microsoft.Extensions.SecretManager.Tests/ConfigurationExtensionTests.cs

```diff
@@ -44,7 +44,7 @@ public void AddUserSecrets_With_An_Existing_Secret_File()
             var projectPath = UserSecretHelper.GetTempSecretProject(out userSecretsId);
```

Comment:
Like I said, I'm fine not doing this now. Are you saying don't bother w/ the Task either? (Fine as well; just not what I expected.)